geode-core/src/main/java/org/apache/geode/internal/cache/PoolFactoryImpl.java

```diff
@@ -104,8 +104,9 @@ public PoolFactory setSocketBufferSize(int bufferSize) {
   }
 
   @Override
+  @Deprecated
```

Comment:
Same.  Already deprecated on the interface API?

Reply:
Same as above.